Overview level (#2139)
giantswarm/docs: src/content/overview/developer portal/_index.md

---
title: Developer portal

> description: An introduction how Giant Swarm uses Backstage to provide a developer portal.

Reviewer: ```suggestion
description: An overview of how to provide a self-service portal to your develpers.
```
